analyses/npl/05_station_correlation.md

```diff
@@ -16,14 +16,16 @@ path_mod = f"{Path(os.path.dirname(os.path.realpath(''))).parents[0]}/"
 sys.path.append(path_mod)
 
 from src.indicators.flooding.glofas import utils, glofas
+
+mpl.rcParams['figure.dpi'] = 300
 ```
 
 ```python
 COUNTRY_ISO3 = 'npl'
 
 DURATION = 1 # Number of days that the GloFAS threshold should be exceeded
 
-RP_LIST = [1.5, 2, 3, 5, 10, 20] # List of return period values to compare
+RP_LIST = [2, 3, 5] # List of return period values to compare
```

Comment:
I think we generally want to stick with 1.5, 2, and 5 year RP because those are the ones GloFAS has marked on their interface (which is what the country office will be using to monitor the forecast)